test/lib/dashboard-agent.spec.js

```diff
@@ -64,18 +66,38 @@ describe("dashboard-agent", function () {
 
       var checkMetrics = function (metrics) {
         expect(metrics).to.be.an("object");
-        expect(metrics.eventLoop).to.deep.equal({ delay: 0, high: 0 });
-        expect(metrics.mem.systemTotal).to.be.above(0);
-        expect(metrics.mem.rss).to.be.above(0);
-        expect(metrics.mem.heapTotal).to.be.above(0);
-        expect(metrics.mem.heapUsed).to.be.above(0);
-        expect(metrics.cpu.utilization).to.be.above(0);
+        expect(metrics.eventLoop.delay).to.be.a("number");
+        expect(metrics.eventLoop.high).to.be.a("number");
+        expect(metrics.mem.systemTotal).to.equal(20);
+        expect(metrics.mem.rss).to.equal(30);
+        expect(metrics.mem.heapTotal).to.equal(40);
+        expect(metrics.mem.heapUsed).to.equal(50);
+        expect(metrics.cpu.utilization).to.equal(60);
       };
 
+      var stubProcess = sinon.stub(process, "memoryUsage", function () {
```

Comment:
Aha, I know what I did wrong, instead of sandbox.stub() I used sinon.stub() and that would explain why it didn't go in the sandbox.

I'll correct this one, thanks!